hits_ingest.py

```diff
@@ -1,60 +1,105 @@
+from lsst.pipe.tasks.ingestCalibs import IngestCalibsArgumentParser
+from lsst.pipe.tasks.processCcd import ProcessCcdTask
+from lsst.pipe.base import ArgumentParser
 from glob import glob
 import sys
-from lsst.obs.decam.ingest import DecamParseTask
+import sqlite3
+import os
 '''
 Little script to ingest some raw decam images
 
-You have to specify two things:
-1. repodir, an empty directory where you want the nicely organized and ingested images to live
-2. datadir, a directory where some '*.fits.fz' images currently live
-
-Usage: $ python hits_ingest.py 'path/to/datadir/'
-
-Result: repodir will be populated with **links** to the files in datadir, organized by date
-
-The command line equivalent with doIngest = True given the variables below is:
-$ ingestImagesDecam.py repodir --filetype raw --mode link datafiles 
-
-And with doIngestCalibs = True, it is (should be?):
-$ ingestCalibs.py repodir --calib calibrepodir --calibType defect --validity 999 datafiles
+PREPARATION
+Make the following empty directories and name them as desired below
+1. repo, where the ingested images and registry will go
+2. calibrepo, where the calibration product registry will go
+
+USAGE
+$ python hits_ingest.py 'path/to/datadir/'
+or
+$ python hits_ingest.py 'path/to/datadir/justsomefiles*.fits.fz'
+(the quotes around the datadir string are important!)
+or
+$ python hits_ingest.py
+(if you don't want to give it an argument because you're running processCcd)
+
+OUTPUT
+if doIngest: repo populated with *links* to files in datadir, organized by date
+             sqlite3 database registry of ingested images also created in repo
+if doIngestCalibs: sqlite3 database registry of ingested calibration products
+                   created in calibrepo
+if doProcessCcd: outputrepo/visit populated with subdirectories containing the
+                 usual post-ISR data (bkgd, calexp, icExp, icSrc, postISR)
+
+BASH EQUIVALENT
+if doIngest:
+    $ ingestImagesDecam.py repo --filetype raw --mode link datafiles
+if doIngestCalibs:
+    $ ingestCalibs.py repo --calib calibrepo --validity 999 datafiles
+if doProcessCcd:
+    $ cd calibrepo
+    $ processCcd.py ../repo --id visit=visit ccdnum=ccdnum 
+            --output ../outputrepo -C $OBS_DECAM_DIR/config/processCcdCpIsr.py
+            --config calibrate.doAstrometry=False calibrate.doPhotoCal=False
+            --no-versions
 '''
-doIngest = True
-doIngestCalibs = False # doesn't work yet, see below
-
-## edit directory names for ingested data repos here ##
-#repodir = '../ingested/' # on lsst-dev
-#calibrepodir = '../calibingested/' # on lsst-dev
-repodir = 'ingested/' # on laptop
-calibrepodir = 'calibingested/' # on laptop
-
-datadir = sys.argv[1] # or '/lsst7/mrawls/HiTS/Blind15A_38/' on lsst-dev or 'data/' on laptop
-datafiles = glob(datadir + '*.fits.fz')
-
-# first, need to make a text file that handles the mapper, per the obs_decam github README
-f = open(repodir+'_mapper', 'w')
+# edit values below as desired
```

Comment:
I think this block below here could basically be replaced with argparse and would be a bit more functional. Plus, you could make the `--help` text print the big message you have above, which would be very handy.